tests/test_rest_api.py

```diff
@@ -154,6 +154,40 @@ def test_liveness_endpoint_wrong_http_method(client):
     assert response.status_code == 405
 
 
+@patch("requests.Session.post", return_value="")
+def test_sync_data1(_mock1, client):
+    """Test the /api/v1/sync-graph-data endpoint."""
+    url = api_route_for("sync-graph-data")
+    inp = {
+        "non_cve_sync": "false",
+        "latest_version_sync": "true",
```

Comment:
@yzainee  inputs for `non_cve_sync` and `latest_version_sync` should be boolean and not string. The same comment applies for the below scenarios as well.